packages/focusjail/src/FocusJailContainer.spec.tsx

```diff
@@ -161,27 +161,6 @@ describe('FocusJailContainer', () => {
         expect(focusSpy).toHaveBeenLastCalledWith(container);
       });
 
-      it("doesn't intercept tab key if not the first or last tabbable item", () => {
```

Comment:
The coverage report indicates this test was not hitting any branches of logic in the source code.

Reply:
Isn't that kind of the point, though? Having a test that proves focus jail is not invoked until necessary seems valuable.

Reply:
I think the logic tested here is specific to `tabbable`. So it wouldn't affect any specific branch coverage.

Reply:
I'm not sure what user-facing behavior this is testing, but that would provide confidence for the `useFocusJail` hook. I'd love to walk through this test tomorrow when we meet.